servers/vultr/controller/main.controller.php

```diff
@@ -377,12 +377,12 @@ private function addVMCustomFields($SUBID)
 			->where('type', 'product')
 			->where('relid', $this->params['packageid'])
 			->where('fieldname', 'LIKE', 'subid|%')->get();
-		if ($customField)
+		if (count($customField) > 0)
```

Comment:
This is not needed and actually breaks the logic flow in things when was debugging this thoroughly today